deployment/network-operator/values.yaml

```diff
@@ -126,15 +126,15 @@ operator:
               - key: "node-role.kubernetes.io/control-plane"
                 operator: In
                 values: [ "" ]
-  repository: nvcr.io/nvidia/cloud-native
+  repository: nvcr.io/nvidia/mellanox
   image: network-operator
-  imagePullSecrets: []
+  #imagePullSecrets: []
   nameOverride: ""
   fullnameOverride: ""
   # tag, if defined will use the given image tag, else Chart.AppVersion will be used
   # tag
 
-imagePullSecrets: []
+#imagePullSecrets: []
```

Comment:
@e0ne does providing no value in image pull secrets here and providing an empty list equivalent when deploying the chart ?

it should be IMO but i want to make sure